openapi/openapi.yaml

```diff
@@ -896,6 +903,7 @@ components:
         [
           is_private,
           system_name,
+          system_tags,
```

Comment:
Since this is a `PATCH` method, I kinda think `system_tags` should be optional. Actually, I think every property should be optional for this endpoint. And our backend should only update the property if it is provided by the user. I noticed that `update_system_by_id()` does not seem to work correctly when `system_details` or `shared_users` is undefined (I think it would assign `None` to those fields if the client does not provide them). Let's merge this as it is and fix `update_system_by_id()` in another PR.